src/components/common/Slider/Slider.tsx

```diff
@@ -0,0 +1,50 @@
+import Slider from '@material-ui/core/Slider'
+import styled from 'styled-components/macro'
+
+interface SliderProps {
+  title: string
+  startValue: number
+  min: number
+  max: number
+  step: number
+  onChange: Function
+}
+
+const SliderComponent = ({
+  title,
+  startValue,
+  min,
+  max,
+  step,
+  onChange,
+}: SliderProps) => {
+  return (
+    <div>
+      <h3>{title}</h3>
+      <StyledSlider
+        defaultValue={startValue}
+        min={min}
+        max={max}
+        step={step}
+        onChange={(event: React.ChangeEvent<{}>, value: number | number[]) =>
+          onChange(value as number)
+        }
+        aria-labelledby="range-slider"
```

Comment:
With aria-labelledby="range-slider" you should set the id of an element that contains the description text for this element. Instead you could make the div a label and replace the h3 with a styled span.

Reply:
Uh thats a nice one, thank you. I could literally hear my brain clicking.